src/RestWrite.js

```diff
@@ -199,9 +201,11 @@ RestWrite.prototype.validateClientClassCreation = function () {
       .then(schemaController => schemaController.hasClass(this.className))
       .then(hasClass => {
         if (hasClass !== true) {
-          throw new Parse.Error(
+          const log = (this.config && this.config.loggerController) || defaultLogger;
```

Comment:
@coderabbitai Why is it only here that we're using the `config.loggerController || defaultLogger`, and not in all other places as well? What's the purpose of this?